### PR TITLE
Update examples to support v1beta3 api - liveness/mysql-wordpress

### PR DESCRIPTION
**Pull request by sub-mod**
@bgrant0607 kindly review

### CODE REVIEW

File: examples/mysql-wordpress-pd/v1beta3/wordpress.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  replicas: 1
+  selector: 
+    name: wordpress
+  template: 
+    metadata: 
+      labels: 
+        name: wordpress
+    spec: 
+      containers: 
+          image: wordpress
```
Comment (bgrant0607) [inline examples/mysql-wordpress-pd/v1beta3/wordpress.yaml]: Needs a `-`

File: examples/mysql-wordpress-pd/v1beta3/mysql.yaml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    name: mysql
+  template: 
+    metadata: 
+      labels: 
+        name: mysql
+    spec: 
+      containers: 
+        - resources:
+            limits :
+              cpu: 1000m
```
Comment (bgrant0607) [inline examples/mysql-wordpress-pd/v1beta3/mysql.yaml]: Same as @vishh's comment on the other PR

**Comment by sub-mod** [comment]
@bgrant0607 i have made the changes, Kindly review.

File: examples/mysql-wordpress-pd/v1beta3/mysql.yaml (hunk 2 above)
Comment (vishh) [inline examples/mysql-wordpress-pd/v1beta3/mysql.yaml]: cpu can be `1` instead of `1000m`.

**Comment by bgrant0607** [comment]
LGTM, thanks.

**Pull request merged**